site/content/entry/yuzu-progress-report-aug-2023/index.md

```diff
@@ -0,0 +1,531 @@
+
+If you experience a game working with no audio, check your drivers, intermediate programs like equalisers, or your OS settings.
+
+Opus is a lossy open-source and royalty-free audio format used by several games.
+Like with the VP9 video format, its implementation on yuzu is not as simple as just playing the file.
+{{< gh-hovercard "11390" "By implementing" >}} the `OpenHardwareOpusDecoderForMultiStreamEx` and `DecodeInterleavedForMultiStream` service methods (Opus multistream support), FearlessTobi ensured games like `MLB The Show 22` are now playable.
+
+{{< gh-hovercard "11419" "Implementing" >}} the `GetWorkBufferSizeExEx` service method, on the other hand, allowed `Sea of Stars` to boot.
+
+{{< single-title-imgs
+    "Nothing beats the look of a classic RPG (Sea of Stars)"
+    "./sos1.png"
+    "./sos2.png"
+    "./sos3.png"
+    "./sos4.png"
+    >}}
+
+If anyone experienced audio related crashes (specifically related to audio effects) in games like `The Legend of Zelda: Tears of the Kingdom`, `SUPER MARIO ODYSSEY` and some others, well, with the help from [ChiefGokhlayeh](https://github.com/ChiefGokhlayeh), Maide {{< gh-hovercard "11408" "found the broken node Id index" >}} that caused the problem.
+
+Still fighting the war of shutting down yuzu in a sensible manner, byte[] changed the {{< gh-hovercard "11384" "shutdown timeout" >}} to more closely match the behaviour of the Switch, with most games now exiting within one second.
+
+Another area where shutting down has improved, this time related to the Skyline framework, is cancelling pending socket operations caused by the TCP logger.
+One of the operations can remain waiting for a host to be accepted. 
+This will never complete once a shutdown event is triggered, so {{< gh-hovercard "11385" "a swift termination is issued" >}} now.
+
+## Android augmentation
+
+Besides all previously reported changes, Android got it’s share of specific bugfixes and UI changes too, courtesy of [t895](https://github.com/t895):
+
+- {{< gh-hovercard "11271" "Miscellaneous settings tweaks," >}} including displaying settings values in list view, reduce the opacity of non-editable settings, for example trying to change resolution multiplier while already running a game, removing some unnecessary default buttons in slider dialogs, for example volume, and a few other minor changes.
+
+{{< imgs
+	"./grey.png|You’re not allowed to change these settings while running a game"
+  >}}
+
+- {{< gh-hovercard "11272" "Show associated values for home settings." >}}
+- {{< gh-hovercard "11273" "Changes to the initial setup," >}} showing if a step was completed, and automatically moving to the next page on completion.
+
+{{< imgs
+	"./setup.gif|Easier setting up!"
```

Comment:
This image should be replaced with a new recording - it is even faster than that now